src/main/webapp/catSortable.js

```diff
@@ -358,7 +358,7 @@ var CategorizedSortable = (function () {
             var sortfn = this.caseInsensitive;
             if (itm.match(/^\d\d[\/-]\d\d[\/-]\d\d\d\d$/)) sortfn = this.date;
             if (itm.match(/^\d\d[\/-]\d\d[\/-]\d\d$/)) sortfn = this.date;
-            if (itm.match(/^[�$]/)) sortfn = this.currency;
+            if (itm.match(/^[£$]/)) sortfn = this.currency;
```

Comment:
Not strictly related to Prototype.js, but side-porting jenkinsci/jenkins#7250 to this copypasta while I was here.